lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -305,23 +324,24 @@ readPoolsPerformances DBLayer{..} (EpochNo epochNo) = do
 -- practice, be greater than 1 if a stake pool produces more than it is
 -- expected.
 calculatePerformance
-    :: Map PoolId (Quantity "lovelace" Word64)
+    :: Int
+    -> Map PoolId (Quantity "lovelace" Word64)
     -> Map PoolId (Quantity "block" Word64)
     -> Map PoolId Double
-calculatePerformance mStake mProd =
+calculatePerformance nTotal mStake mProd =
     let
         stakeButNotProd = traverseMissing $ \_ _ -> 0
         prodButNoStake  = dropMissing
-        stakeAndProd sTotal nTotal = zipWithMatched $ \_ s n ->
+        stakeAndProd sTotal = zipWithMatched $ \_ s n ->
             if (nTotal == 0 || s == Quantity 0) then
                 0
             else
-                min 1 ((double n / nTotal) * (sTotal / double s))
+                ((double n / fromIntegral nTotal) * (sTotal / double s))
```

Comment:
The cap will however still be necessary to cope with the case where pools are elected for more slots than expected.

Reply:
I'm not _completely_ sure it's desirable (since it "destroys information"), but I re-added it 🤔 🤷‍♂ 👍